pkg/ovsdb/client/client.go

```diff
@@ -75,7 +75,7 @@ func NewOvsDbClient(db, addr string, dbModel model.ClientDBModel, monitors []cli
 		tlsConfig := &tls.Config{
 			Certificates:       []tls.Certificate{cert},
 			RootCAs:            certPool,
-			InsecureSkipVerify: true,
+			InsecureSkipVerify: false,
```

Comment:
Why change this value?

Reply:
oh，he changes this because of a lint err. will remove later. 

the `// #nosec` before this line has no effect.

Reply:
I will checkout this change